en/test_and_ci/docker.md

```diff
@@ -126,13 +126,17 @@ xhost +
 # Run docker and open bash shell
 docker run -it --privileged \
 --env=LOCAL_USER_ID="$(id -u)" \
--v ~/src/PX4-Autopilot:/src/PX4-Autopilot/:rw \
+-v ~/Projects/PX4-Autopilot:/src/PX4-Autopilot/:rw \
```

Comment:
This bit is wrong. The bit you changed is the `<host_src>` and the example above explicitly states "share the directory **~/src/PX4-Autopilot** on the host computer"

Reply:
I forgot to change that back, I'll go ahead and fix that.